letsencrypt-nginx/letsencrypt_nginx/configurator.py

```diff
@@ -213,17 +216,13 @@ def choose_vhost(self, target_name):
         :rtype: :class:`~letsencrypt_nginx.obj.VirtualHost`
 
         """
-        vhost = None
-
         matches = self._get_ranked_matches(target_name)
         if not matches:
-            # No matches. Create a new vhost with this name in nginx.conf.
-            filep = self.parser.loc["root"]
-            new_block = [['server'], [['server_name', target_name]]]
-            self.parser.add_http_directives(filep, new_block)
-            vhost = obj.VirtualHost(filep, set([]), False, True,
-                                    set([target_name]), list(new_block[1]))
-        elif matches[0]['rank'] in xrange(2, 6):
+            logger.info("Virtual Host not found for %s", target_name)
```

Comment:
:+1: 
Make sense for me also not looking to the future, the phrase "Are you sure you're really serving that domain name?" for me, provide a better user feedback :)